### PR TITLE
Add support in wps for retrieving data on local filesystem

### PR DESCRIPTION
**Pull request by huard**
Fix for #680 

Includes test.

### CODE REVIEW

File: owslib/wps.py [hunk 1]
```diff
@@ -1430,6 +1430,9 @@ def retrieveData(self, username=None, password=None, headers=None, verify=True,
                         headers=headers, verify=verify, cert=cert)
             # extract output filepath from URL query string
             self.fileName = spliturl[1].split('=')[1]
+        elif url.startswith("file://"):
+            u = open(url[7:])
```
Comment (tomkralidis) [inline owslib/wps.py:1434]: Would this result in an open file handle ?  Would adding a context manager help here (this block would need some logic rework) https://github.com/geopython/OWSLib/blob/c0d83476e91cf7ef0ec82134ffb828f95685ad06/owslib/wps.py#L1426-L1444)

Comment (huard) [reply owslib/wps.py]: Not sure. Let's not take any chance.

**Comment by cehbrecht** [comment]
@huard there is a common wps test:
https://github.com/geopython/OWSLib/blob/master/tests/test_wps.py

... and offline tests with predefined wps responses, like:
https://github.com/geopython/OWSLib/blob/master/tests/test_wps_execute.py

**Comment by huard** [comment]
@cehbrecht  Test added.

**Comment by cehbrecht** [comment]
@huard merged. Thanks :)

**Pull request merged**